website/docs/operations.md

```diff
@@ -176,6 +175,28 @@ At a high level, this requires:
 
 * The ability to read all objects in the group `mutations.gatekeeper.sh` (mutators)
 
+## Generation
+
+__--operation key:__ `generate`
+
+This operation enables CRD and VAP/VAPB generation.
```

Comment:
```suggestion
This operation enables CRD and VAP/VAPB generation. 

To avoid write contention, the Generate operation should be run as a singleton.
```

Reply:
I think we are missing the operation here: https://github.com/open-policy-agent/gatekeeper/blob/master/manifest_staging/deploy/gatekeeper.yaml#L5084-L5086

Reply:
Good catch! I fixed it. Do we want to cut 3.18.1 to update `deploy/gatekeeper.yaml`?